openpype/hosts/maya/api/workfile_template_builder.py

```diff
@@ -341,6 +342,10 @@ def update_workfile_template(*args):
     builder.rebuild_template()
 
 
+def open_template(*args):
```

Comment:
To be honest this function name seems to be misleading. It does not direct to which template it opens and completelly ignores fact that it requires user interaction because of UI.

Does it have to be public function that is here, instead of private function in pipeline.py (or where menu is built)?

Reply:
Good point!